spacy/errors.py

```diff
@@ -856,6 +856,8 @@ class Errors:
     E1017 = ("A Doc object requires both 'deps' and 'heads' for dependency "
              "parses. If no dependency labels are available, provide "
              "placeholder deps such as `deps=[\"dep\"]*len(heads)`.")
+    E1018 = ("Knowledge base for component '{name}' is not set. "
+             "Use the methods `kb.add_entity` and `kb.add_alias` to add entries.")
```

Comment:
```suggestion
    E1018 = ("Knowledge base for component '{name}' is not set. "
             "Make sure either `nel.initialize` or `nel.set_kb` is called with a `kb_loader` function.")
```

Reply:
Yeah, this error message is much more helpful. Thanks!